themes/custom/essex/css/overrides.css

```diff
@@ -1328,24 +1328,6 @@ tbody tr {
   padding: var(--spacing-larger) 0 var(--spacing);
 }
 
-
```

Comment:
Does this affect news article teaser in components?

Reply:
I think it's ok, what do you think @kurtisRogers-Nomensa:

![image](https://github.com/essexcountycouncil/essex-intranet-drupal/assets/158008/8383fe61-6eae-4f9f-adeb-a077072a5b0f)